src/widgets/settingspages/HighlightingPage.cpp

```diff
@@ -209,73 +278,92 @@ HighlightingPage::HighlightingPage()
 
     // ---- misc
     this->disabledUsersChangedTimer_.setSingleShot(true);
-}  // namespace chatterino
+}
```

Comment:
Why was the namespace comment removed from here?
```suggestion
} // namespace chatterino
```

Reply:
This had been accidentally added at some point, so this is just a minor unrelated fix